languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8788,6 +8788,20 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="LA_OU_LES_PLUS_DE_NOM" name="la ou les plus nom" default="temp_off">
+          <pattern>
+              <marker>
+                  <token regexp="yes">la|les</token>
+              </marker>
+              <token regexp="yes">plus|moins</token>
+              <token regexp="yes">de</token>
+              <token postag="N.*" postag_regexp="yes"></token>
+          </pattern>
+          <message>«la» est une expression fautive.</message>
```

Comment:
Could you make this message more helpful? "La" is not really an expression... how about something along the lines of `Dans l'expression 'le \2 \3', l'article reste toujours <suggestion>le</suggestion>.`?